pyproject.toml

```diff
@@ -53,10 +53,18 @@ black = "=22.12.0"
 isort = "^5.11.4"
 
 # Linters
-mypy = "*"
 pydocstyle = "^6.3.0"
 pylint = "^2.16.2"
 
+# FIXME: Upgrading mypy will require updates to aliased fields. e.g.
```

Comment:
i noticed this too, I think these aliases are just for serialization, and updating to `serialization_alias` makes mypy happy (also see: https://github.com/anvilco/python-anvil/pull/65/files)